boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad_iron_0_9_0.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54h20dk/nrf54h20/cpurad/iron
+name: nRF54H20-DK-nRF54H20-Radio (IRONside SE compatible) (revision 0.9.0)
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+sysbuild: true
+ram: 192
+flash: 256
```

Comment:
This is 336 kB in `nrf54h20dk_nrf54h20-memory_map_iron.dtsi`

Reply:
I have a patch for this. But i'd rather apply this after this PR goes in.

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/89333